src/helpers/dataCreator.ts

```diff
@@ -34,5 +28,5 @@ export const getEmptyOptions = (count: number) =>
     .map((val, idx) => ({
       optionId: nanoid(),
       text: '',
-      isCorrect: false,
+      isCorrect: idx === 0,
```

Comment:
Why do we want the 1st one selected?
Let the user decide which they want to select.
They might by mistake save with the 1st one selected which might not be correct.